src/applications/personalization/dashboard/components/Dashboard.jsx

```diff
@@ -327,14 +320,8 @@ const mapStateToProps = state => {
     hasClaimsOrAppealsService;
   const showHealthCare =
     !showMPIConnectionError && !showNotInMPIError && isLOA3 && isVAPatient;
-  const signInService = signInServiceNameSelector(state);
   const showBenefitPaymentsAndDebt =
-    toggleValues(state)[FEATURE_FLAG_NAMES.showPaymentAndDebtSection] &&
-    !showMPIConnectionError &&
-    !showNotInMPIError &&
-    isLOA3 &&
-    signInServicesEligibleForDD.has(signInService) &&
-    isMultifactorEnabled(state);
```

Comment:
Double checked to see that this is indeed what we want to remove as far as check, and we think that is the case.